vexim/config/httpheaders.php

```diff
@@ -1,3 +1,4 @@
+<!DOCTYPE html>
```

Comment:
Adding a doctype before headers are sent will cause problems with output buffering disabled.
If you really want it, I think you should add it before the `<html>` tag in all pages, although I wonder if it makes any difference with the sort of HTML we're talking about.
And I don't think you can add it after the `header()` lines here as well, because this file is included in `FOOsubmit.php` files as well, which might send their own headers.